WooCommerce/Classes/ViewRelated/Orders/OrderDetailsViewController.swift

```diff
@@ -504,6 +589,23 @@ private extension OrderDetailsViewController {
         let fulfillViewController = FulfillViewController(order: viewModel.order)
         navigationController?.pushViewController(fulfillViewController, animated: true)
     }
+
+    func trackingWasPressed(at indexPath: IndexPath) {
+        guard let tracking = orderTracking(at: indexPath) else {
+            return
+        }
+
+        guard let trackingURL = tracking.trackingURL, let url = URL(string: trackingURL) else {
+            return
+        }
+
```

Comment:
We should add a `WooAnalytics.shared.track(.orderDetailTrackShipmentButtonTapped)` call here. Details coming to you on Slack 😄